lambda/MAS/Services/StaticWebsiteService.cs

```diff
@@ -28,55 +30,90 @@ public class S3StaticWebsiteService : IStaticWebsiteService
         private readonly IAmazonS3 _amazonS3;
         private readonly ILogger<S3StaticWebsiteService> _logger;
         private readonly AWSConfig _awsConfig;
+        private readonly IAmazonCloudFront _cloudFrontService;
+        private readonly EnvironmentConfig _environmentConfig;
+        private readonly CloudFrontConfig _cloudFrontConfig;
 
-        public S3StaticWebsiteService(IAmazonS3 amazonS3, ILogger<S3StaticWebsiteService> logger, AWSConfig awsConfig)
+        public S3StaticWebsiteService(IAmazonS3 amazonS3, 
+            IAmazonCloudFront cloudFrontService, 
+            ILogger<S3StaticWebsiteService> logger,
+            AWSConfig awsConfig, 
+            EnvironmentConfig environmentConfig,
+            CloudFrontConfig cloudFrontConfig)
         {
             _amazonS3 = amazonS3;
             _logger = logger;
             _awsConfig = awsConfig;
+            _cloudFrontService = cloudFrontService;
+            _environmentConfig = environmentConfig;
+            _cloudFrontConfig = cloudFrontConfig;
         }
 
         #endregion
 
-        public async Task<HttpStatusCode> WriteFileAsync(string filePath, string contentBody)
+
+        public async Task<HttpStatusCode> WriteFilesAsync(params StaticContentRequest[] requests)
         {
-            PutObjectRequest request = new PutObjectRequest()
-            {
-                BucketName = _awsConfig.BucketName,
-                Key = filePath,
-                ContentBody = contentBody
-            };
+            HttpStatusCode responseCode = HttpStatusCode.OK;
+            var taskDict = new Dictionary<StaticContentRequest, Task<PutObjectResponse>>();
+            foreach(var req in requests)
+                taskDict.Add(req, WriteFileAsync(req));
 
-            try
+            foreach(var task in taskDict)
             {
-                var response = await _amazonS3.PutObjectAsync(request);
-                return response.HttpStatusCode;
-            }
-            catch (Exception e)
-            {
-                _logger.LogError(e, $"Failed to write ${filePath} to S3: {e.Message}");
-                throw e;
+                responseCode = (await task.Value).HttpStatusCode;
+                if (responseCode != HttpStatusCode.OK)
+                {
+                    _logger.LogError($"Writing {task.Key.FilePath} resulted in a status code of {responseCode}");
+                    return responseCode;
+                }
             }
+
+            if (_cloudFrontConfig.Enabled.ToLower() == "true" && requests.Count() > 0)
```

Comment:
Yeah it should def be true, cos then you don't need to ToString it here
```suggestion
            if (_cloudFrontConfig.Enabled && requests.Count() > 0)
```